blog/incident-report-week3/index.qmd

```diff
@@ -0,0 +1,30 @@
+---
+author: Mordred Boulais
+title: Incident Report Regarding First Professional Precis
+date: '2023-9-11'
+date-format: YYYY
+categories: [post, developer development, incident report, professional precis, SE1]
+description: <em>A discussion of what we struggled with and how to improve from our first incident.</em>
```

Comment:
Make sure that this fits on a single line in both the blog post listing and the page for this blog post itself.